### DISCUSSION TITLE
How to stop Emoji from being wrapped in Mark ？

### DISCUSSION THREAD

**Discussion by Alecyrus** [ueberdosis/tiptap#2206] [General Questions] [answered]
`Emoji Node` is not compatible with my custom mark, so I want to add `marks: ""` to the nodeSpec of `Emoji Node`. And I try the following code but does not work. Anybody help?

```javascript
 Emoji.extend({
    marks: "_"
  }).configure({
    emojis: gitHubEmojis,
  })
```

**Comment by philippkuehn** [comment]
what do you mean with "not compatible" or "does not work"?

**Reply by Alecyrus** [reply]
"not compatible":   the style of Mark should not be applied to emoji, which will cause unexpected render result.
"does not work": I add `marks:""` to NodeSpec of Emoji Node to disallow all marks, but marks still can be added to the emoji node.

**Comment by Alecyrus** [comment, marked as answer]
[Disallowing marks on some nodes isn’t supported](https://discuss.prosemirror.net/t/dont-wrap-certain-inline-nodes-in-another-marks/2784), so sad.

but I solved the problem by applying a specific style to the emoji node to override Mark's style
